### PR TITLE
Fix bar chart center label

### PR DESCRIPTION
**Pull request by fgervasi**
### Issue
https://3.basecamp.com/3500782/buckets/1333707/todos/2480040725

Example:
- https://q.st-staging.nzz.ch/item/e9046b127bd99afc9cd208b94d1b2310
- https://q.st-staging.nzz.ch/item/e9046b127bd99afc9cd208b94d1b17cd
- https://q.st-staging.nzz.ch/item/e9046b127bd99afc9cd208b94d1a5183

### CODE REVIEW

**Comment by manuelroth** [comment]
This looks good to me. Could you also apply this change to stacked bar-charts?- I think this problem is visible there as well: https://q.st-test.nzz.ch/item/chart-29

File: chartTypes/bar/mapping.js [hunk 1]
```diff
@@ -190,7 +190,7 @@ module.exports = function getMapping() {
                 field: "y"
               },
               dy: {
-                signal: "barWidth / 2"
+                signal: "(barWidth / 2) + 0.5", // +0.5 Fix font is not exact in the middle of bar
```
Comment (manuelroth) [inline chartTypes/bar/mapping.js:193]:
```suggestion
                signal: "(barWidth / 2) + 0.5", // Adds offset of 0.5px in order to vertically center the label within the bar
```

I think this comment is a bit more clear. What do you think?

**Comment by fgervasi** [comment]
By the stacked bar it is not so nice that the bar only goes up to halfway when a value line is adjacent. However, it always look the same as the current version.

The solution doesn't seem that simple. It would probably be worth opening a new issue. what do you mean ?
image

**Review by manuelroth — APPROVED** [review]

**Comment by manuelroth** [comment]
Yes, let's open a new issue for this problem 👍

**Pull request merged**